op-test-sequencer/sequencer/service.go

```diff
@@ -67,11 +67,14 @@ func (s *Service) initFromCLIConfig(ctx context.Context, cfg *config.Config) err
 	if err := s.initMetricsServer(cfg); err != nil {
 		return fmt.Errorf("failed to start Metrics server: %w", err)
 	}
+	if err := s.createRPCHandler(cfg); err != nil {
```

Comment:
`s.backend` references `s.rpcHander`, and the other way around, so `initRPCHandler` had to be split.